stripe/src/main/java/com/stripe/android/model/PaymentMethodOptionsParams.kt

```diff
@@ -39,4 +39,19 @@ sealed class PaymentMethodOptionsParams(
             private const val PARAM_NETWORK = "network"
         }
     }
+
+    @Parcelize
+    data class Blik(
+        var code: String? = null,
+    ) : PaymentMethodOptionsParams(PaymentMethod.Type.Blik) {
+        override fun createTypeParams(): List<Pair<String, Any?>> {
+            return listOf(
+                PARAM_CODE to code,
+            )
+        }
+
+        companion object {
+            const val PARAM_CODE = "code"
+        }
```

Comment:
`private companion object`

Reply:
made this package private to that [test](https://github.com/stripe/stripe-android/pull/3479/files#diff-c49582ec19f521759fc91e54b8e3cdeab02022b84fd3423311cd80b91aead4aeR33) could access them, please lmk if this is necessary, happy to change otherwise :)

Reply:
lgtm!